### PR TITLE
To fix restore issues (partial commit and other problems)

### PR DESCRIPTION
**Pull request by sdminonne**
@cdoan1 @andreadecorte @zkayyali812 here is a working restore

### CODE REVIEW

File: ansible-playbooks/roles/connect-aks-to-acm-hub-collection/tasks/delete.yml [hunk 1]
```diff
@@ -25,7 +48,8 @@
 
 - name: Get network interfaces within a resource group
   azure.azcollection.azure_rm_networkinterface_info:
-    resource_group: "{{ AKS_MRG }}"
+    resource_group: "{{ AKS_VNET_RG
```
Comment (cdoan1) [inline ansible-playbooks/roles/connect-aks-to-acm-hub-collection/tasks/delete.yml:51]: typo here ?

Comment (sdminonne) [reply ansible-playbooks/roles/connect-aks-to-acm-hub-collection/tasks/delete.yml:51]: Clear... But I tested multiple time and it worked... Please don't merge

Comment (sdminonne) [reply ansible-playbooks/roles/connect-aks-to-acm-hub-collection/tasks/delete.yml:51]: Ah ok. 
`- hosts: localhost
  vars:
     myvar_es: "mundo"
     myvar_us: "world"
  connection: local
  gather_facts: no
  become: false
  tasks:
  - name: this is my test
    debug:
      msg: "hello {{ myvar_es
}} "
`

results in:
`$ ansible-playbook  play.yml 
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

PLAY [localhost] **********************************************************************************************************************************************************************************************************************************************************************************************************

TASK [this is my test] ****************************************************************************************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "hello mundo "
}

PLAY RECAP ****************************************************************************************************************************************************************************************************************************************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

`

Comment (sdminonne) [reply ansible-playbooks/roles/connect-aks-to-acm-hub-collection/tasks/delete.yml:51]: it looks like the `\n` is skipped inside a `jinjavar`

Comment (sdminonne) [reply ansible-playbooks/roles/connect-aks-to-acm-hub-collection/tasks/delete.yml:51]: :) finally it's a nit :) But I fix it anyway

File: ansible-playbooks/roles/restore-hub/tasks/main.yml [hunk 2]
```diff
@@ -5,12 +5,38 @@
...
+      kind: MultiClusterHub
+      metadata:
+        name: multiclusterhub
+        namespace: open-cluster-management
+      spec:
+        enableClusterBackup: true
+
+- name: Let's pause 1m to let cluster-backup operator run
+  wait_for:
+    timeout: 60
```
[the remark this answers is not in the document]
Comment (sdminonne) [reply ansible-playbooks/roles/restore-hub/tasks/main.yml:21]: Absolutely I'm working on it

File: ansible-playbooks/roles/restore-hub/tasks/main.yml [hunk 3]
```diff
@@ -25,22 +51,40 @@
...
-- name: cleanup AKS
+- name: deconnect AKS
   include_tasks: ../roles/connect-aks-to-acm-hub-collection/tasks/delete.yml
   vars:
-     AKS_NAME:  "{{  item.ID.split('/')[8] }}"
-     AKS_MRG: "{{ item.ID.split('/')[4] }}"
-  with_items:
-    - "{{ aks_ids }}"
-  when: ( item.ID.split('/')[8] in {{ mc_lists }} )
+     AKS_NAME:  "{{  aks_item.ID.split('/')[8] }}"
```
[the remark this answers is not in the document]
Comment (sdminonne) [reply ansible-playbooks/roles/restore-hub/tasks/main.yml:57]: Good to know. I'm pretty new to Ansible... Keep learning. Thanks!

**Comment by sdminonne** [comment]
@andreadecorte @cdoan1 if you think this is mergeable (somehow) may we go ahead and merge? For sure I'll work on Andrea suggestion about _custom filter_ and ~~wait~~ -> `check status`

**Review by cdoan1 — APPROVED** [review]
LGTM

**Pull request merged**